.github/workflows/jib-cli-release.yml

```diff
@@ -46,8 +46,8 @@ jobs:
       - name: Create pull request
         uses: repo-sync/pull-request@v2
         with:
-          # Provided by Actions. No need to create a token.
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Use a personal access token instead of the one created by Github Actions.
+          github_token: ${{ secrets.GA_RELEASE_PR_PERSONAL_TOKEN }}
           source_branch: cli-release-v${{ github.event.inputs.release_version }}
           pr_title: "CLI release v${{ github.event.inputs.release_version }}"
           pr_body: "To be merged after the release is complete."
```

Comment:
Remove the kokoro label below.

Reply:
Oh, we should also add `PR: Merge After Release` too.

Reply:
oops nice catch